Pass args to ack/nack

sebastianandreasson opened the pull request.

sebastianandreasson committed:
pass args with ack/nack

sebastianandreasson committed:
Merge remote-tracking branch 'JohanObrink/master'

JohanObrink commented:
Could you add a test that verifies that the passed parameters work and one to confirm that leaving the parameters out doesn't break anything?

sebastianandreasson committed:
update integration tests

Also updated the exchange/queuenames for all the tests, it caused
conflicts for me when testing because rabbit thought the same exchange
was ‘direct’ from previous test when the next wanted it as ‘fanout’ for
example.

Having unique names solved this issue for me.

JohanObrink commented:
Thanks man!
